src/Lib/Managers/ItemsManager.vala

```diff
@@ -564,6 +563,56 @@ public class Akira.Lib.Managers.ItemsManager : Object, Items.ModelListener {
         }
     }
 
+    public void create_group_from_selection () {
```

Comment:
@mbfraga Initial rough implementation of what I think I figured it should happen.
TLDR:
- Collect all selected items in a tree map sorted by pos key.
- Create the group and add it to the canvas.
- Loop through the sorted tree and clone instance, delete node, add instance to the group.

Does this make sense or am I doing something incredibly silly?

Reply:
I think it would make sense to use a Util here to ensure the new node is an exact copy of the old node, maintaining its name and all its properties.
Unless, this is overkill and the only thing needed here is changing the parent ID (not sure if that would work), like a proper "move this item into this container", which will be the same method used to move items between artboards.

Reply:
This method will most likely leverage the `Model::transfer()` method I'm slowly adding.